apps/web/lib/team/[slug]/[type]/getServerSideProps.tsx

```diff
@@ -53,6 +53,9 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
       ...getSlugOrRequestedSlug(teamSlug),
       parent: isValidOrgDomain && currentOrgDomain ? getSlugOrRequestedSlug(currentOrgDomain) : null,
     },
+    orderBy: {
+      slug: { sort: "asc", nulls: "last" },
+    },
```

Comment:
We need to prioritize checking the slug before the requestSlug, as those are unpublished

Reply:
My intuition is this is just hiding a deeper problem.

Perhaps we change this `where` to make sure the team/org is verified and paid.